PWGCF/FemtoDream/Core/femtoDreamDetaDphiStar.h

```diff
@@ -372,16 +457,64 @@ class FemtoDreamDetaDphiStar
     return phiAtRadii;
   }
 
+  template <typename T>
+  int PhiAtRadiiTPCForHF(const T& part, std::vector<float>& tmpVec, int prong)
+  {
+    int charge = 0;
+    float pt = -999.;
+    float phi0 = -999.;
+    switch (prong) {
+      case 0:
+        pt = part.prong0Pt();
+        phi0 = part.prong0Phi();
+        charge = part.charge();
+        break;
+      case 1:
+        pt = part.prong1Pt();
+        phi0 = part.prong1Phi();
+        charge = -part.charge();
+        break;
+      case 2:
+        pt = part.prong2Pt();
+        phi0 = part.prong2Phi();
+        charge = part.charge();
+        break;
+      default:
```

Comment:
Please use meaningful constants instead of numbers
It is a good approach defining them within an `enum`